vars/buildPlugin.groovy

```diff
@@ -303,11 +277,7 @@ def call(Map params = [:]) {
                     publishingIncrementals = true
                   } else {
                     String artifacts
-                    if (isMaven) {
-                      artifacts = '**/target/*.hpi,**/target/*.jpi,**/target/*.jar'
-                    } else {
-                      artifacts = '**/build/libs/*.hpi,**/build/libs/*.jpi'
-                    }
+                    artifacts = '**/target/*.hpi,**/target/*.jpi,**/target/*.jar'
                     archiveArtifacts artifacts: artifacts, fingerprint: true
```

Comment:
The only reason to pull this out into a variable would have been to support the conditional structure that is now being deleted. Now that the conditional logic is being deleted, leaving behind a variable that is only ever used one time does not make sense and only serves to make the code less readable by adding an unnecessary layer of indirection. In general, any variable that is used only one time should be inlined unless there is a specific reason otherwise.

```suggestion
                    archiveArtifacts artifacts: '**/target/*.hpi,**/target/*.jpi,**/target/*.jar', fingerprint: true
```